apps/prairielearn/src/lib/question.js

```diff
@@ -1399,9 +1399,10 @@ module.exports = {
       urls.calculationQuestionFileUrl = questionUrl + 'file';
 
       // FIXME: broken?
-      urls.calculationQuestionGeneratedFileUrl = questionUrl + 'generatedFilesQuestion';
+      urls.calculationQuestionGeneratedFileUrl =
+        questionUrl + 'generatedFilesQuestion/variant/' + variant.id;
 
-      urls.clientFilesCourseUrl = urlPrefix + '/clientFilesCourse';
+      urls.clientFilesCourseUrl = questionUrl + 'clientFilesCourse';
```

Comment:
Feels like we are missing a `/` here? Also in the line below in this file.

Reply:
Look a few lines up - `questionUrl` is defined with a trailing slash. Ditto for below.

Reply:
Ah! Apologies, I had misread a wrapped line.